### PR TITLE
added image component

### PR DESCRIPTION
**Pull request by raieegulhane**
Component variants:
- [ ]  responsive images
- [ ]  various appearances
- [ ]  bordered images

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
Good to go.

File: components/images/images.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+.img-50pr {
+    width: 50%;
+}
+
+.img-25pr {
+    width: 25%;
+}
+
+.img {
```
Comment (Manishsharma203) [inline components/images/images.css:17]: Looks good. Is there any specific use case of this class with fixed height and width?

**Review by sakshi006 — APPROVED** [review]
Looks awesome!
Can merge the PR.

File: components/images/images.css (hunk 1 above)
Comment (raieegulhane) [reply components/images/images.css:17]: yes. For the display of image in that gray box on component page.  Round image will need equal height and width.

**Pull request merged**